src/components/Dashboard.vue

```diff
@@ -635,6 +636,16 @@ export default Vue.extend({
     naIfEmptyStringArray(arr: readonly string[]) {
       return arr && arr.length !== 0 && arr[0] !== '' ? arr : ['N/A'];
     },
+
+    deleteCourseFromSemesters(subject: string, number: string) {
+      const updatedSemesters = this.semesters.map(semester => {
+        const coursesWithoutDeleted = semester.courses.filter(
+          course => course.subject !== subject && course.number !== number
+        );
+        return { ...semester, courses: coursesWithoutDeleted };
+      });
+      this.editSemesters(updatedSemesters);
+    },
```

Comment:
For the case of duplicate non AP/IB credits (e.g. having CS 1110 twice in the schedule), maybe we can check the `uniqueID` of the course in addition to the `subject` and `number`. This may involve adding a `courseUniqueID` data field in `CompletedSubReqCourse.vue`.

Reply:
Yea so I think we'll need support for AB/IB credits first? But it would definitely be for another PR.